tvtk/tests/test_visual.py

```diff
@@ -255,6 +260,8 @@ def test_helix(self):
         h.axis = 0., 1., 0.
         # Then
         bounds = get_bounds((1.0, 1.5, 1.0), (0.4, 1.0, 0.4))
+        if vtk_major_version >= 9:  # XXX probably a bug somewhere
+            bounds = [1.02, 2., 0.809789, 1.190211, 0.8, 1.2]
```

Comment:
Yes, this is showing that box is broken, so please remove this too.  I can fix this later -- will put it on my todo.

Reply:
BTW it would be really nice to do:
```
@pytest.mark.xfail(vtk_major_version >= 9, reason='Broken on VTK9')
```
In principle it's as easy as installing and using `pytest` instead of `nose`, which is unmaintained (AFAIK) nowadays anyway. Let me know if you want me to push a PR to transition to `pytest` so that future code can make use of it.

Reply:
I actually would like that as I use pytest in all my other projects.  I think ETS projects do not use pytest. 
@mdickinson and @corranwebster  -- is it OK to use pytest for ETS projects?

Reply:
I guess there are two ways to use `pytest`: as a test runner (which is fine), or as a test framework. At the moment, the ETS test suites (at least for core packages, like TraitsUI, Pyface, Envisage, Traits) can be run using any of `unittest`, `nose` or `pytest`, and I'd really like to keep it that way if possible.

This could be a `skipIf` in `unittest`, or possibly an `expectedFailure` if you still want to run the test code with VTK >= 9.0, but are still expecting it to fail.